assets/dev/js/editor/document/hooks/ui/create/move-column-resize-handle.js

```diff
@@ -0,0 +1,48 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class MoveColumnResizeHandle extends After {
+	getCommand() {
+		return 'document/elements/create';
+	}
+
+	getId() {
+		return 'move-column-resize-handle--document/elements/create';
+	}
+
+	getContainerType() {
+		return 'column';
+	}
+
+	getConditions() {
+		// Trigger the hook only if DOM optimization experiment is active.
+		return elementorCommon.config.experimentalFeatures[ 'e_dom_optimization' ];
+	}
+
+	apply( args ) {
+		const { containers = [ args.container ] } = args;
+
+		containers.forEach( ( /* Container */ container ) => {
+			if ( 'column' === container.model.get( 'elType' ) ) {
```

Comment:
You already filtered to columns only by `getContainerType`.

Reply:
Yup, right :)